apps/api/src/test/java/dk/treecreate/api/order/OrderServiceTest.java

```diff
@@ -254,33 +257,64 @@ void calculateTotal(BigDecimal subTotal, int discountAmount, DiscountType discou
         }
     }
 
-    private static Stream<Arguments> pricePerItemArguments()
-    {
+    private static Stream<Arguments> pricePerItemArguments() {
         return Stream.of(
-            Arguments.of(DesignType.FAMILY_TREE, DesignDimension.SMALL, new BigDecimal(495), null),
-            Arguments.of(DesignType.FAMILY_TREE, DesignDimension.MEDIUM, new BigDecimal(695), null),
-            Arguments.of(DesignType.FAMILY_TREE, DesignDimension.LARGE, new BigDecimal(995), null),
-            Arguments.of(DesignType.FAMILY_TREE, DesignDimension.ONE_SIZE, null,
-                "dimension data is not valid")
-        );
+                Arguments.of(DesignType.FAMILY_TREE, DesignDimension.SMALL, new BigDecimal(495), null),
+                Arguments.of(DesignType.FAMILY_TREE, DesignDimension.MEDIUM, new BigDecimal(695), null),
+                Arguments.of(DesignType.FAMILY_TREE, DesignDimension.LARGE, new BigDecimal(995), null),
+                Arguments.of(DesignType.FAMILY_TREE, DesignDimension.ONE_SIZE, null,
+                        "dimension data is not valid"));
     }
 
     @ParameterizedTest
     @MethodSource("pricePerItemArguments")
     @DisplayName("pricePerItem() correctly returns price based on design type and dimensions")
     void pricePerItem(DesignType designType, DesignDimension designDimension,
-                      BigDecimal expectedPrice, String exceptionMessageSnippet)
-    {
-        // there is no point testing the default options sine they are impossible to each as of now (there are not enum values not covered by the switches)
-        if (expectedPrice != null)
-        {
+            BigDecimal expectedPrice, String exceptionMessageSnippet) {
+        // there is no point testing the default options sine they are impossible to
+        // each as of now (there are not enum values not covered by the switches)
+        if (expectedPrice != null) {
             assertEquals(orderService.pricePerItem(designType, designDimension), expectedPrice);
-        } else
-        {
+        } else {
             Throwable exception = assertThrows(ResponseStatusException.class, () -> {
                 orderService.pricePerItem(designType, designDimension);
             });
             assertThat(exception.getMessage()).contains(exceptionMessageSnippet);
         }
     }
+
+    @Test
+    @DisplayName("updateOrderStatus() correctly updates the status of the order")
+    void updateOrderStatus() {
+        // prepare the order
+        UUID orderId = UUID.fromString("c0a80121-7adb-10c0-817a-dbc2f0ec1234");
+        OrderStatus status = OrderStatus.ASSEMBLING;
+        OrderStatus newStatus = OrderStatus.ASSEMBLING;
```

Comment:
Since the statuses are the same, the test can't fully verify that the service does actually update them
```suggestion
        OrderStatus newStatus = OrderStatus.NEW;
```